ledger-api/grpc-definitions/BUILD.bazel

```diff
@@ -70,69 +73,115 @@ proto_gen(
     ],
 )
 
-# TODO(nic): When compile-proto-file handles map<key,value>, we will can run on:  **/*.proto
-# + glob(["**/*.proto",])
+google_protobuf_src = "external/com_google_protobuf/src"
+
+genrule(
+    name = "google-protobuf-haskellpb-sources",
+    srcs = ["@com_google_protobuf//:well_known_protos"],
+    outs = ["Google/Protobuf/" + b for b in [
+        "Any.hs",
```

Comment:
You could probably factor out the duplication between the list here and the list in the cmd.

Reply:
tricky because of the mismatch between CamlCase and snake_case